NOTICE.md

```diff
@@ -1,4 +1,4 @@
-# Notices for Eclipse Project for Stable Jakarta EE APIs
+# Notices for  Jakarta Stable APIs
 
 This content is produced and maintained by the Eclipse Project for Stable
```

Comment:
Update project name here also.  I didn't look very closely, did we also miss these on the management repo?

Reply:
Yes... I'll fix that too.